src/helpers/accessiblity.ts

```diff
@@ -59,6 +59,11 @@ export function isHiddenFromAccessibility(
 export const isInaccessible = isHiddenFromAccessibility;
 
 function isSubtreeInaccessible(element: ReactTestInstance): boolean {
+  // Null props can happen for React.Fragments
+  if (element.props == null) {
```

Comment:
can it be undefined as well? if so, let's go with `===`

Reply:
`== null` is a safer check, as it matches exactly `undefined` and `null`.

Reply:
BTW according to official type info this cannot be `undefined` but is for `React.Fragments`

Reply:
Yeah this is what I asked for, just a bit indirectly. Let's leave it as is then